x/incentives/keeper/proposals.go

```diff
@@ -0,0 +1,69 @@
+package keeper
```

Comment:
@fedekunze How can I call the mint denomination?

Reply:
`k.mintKeeper.GetParams(ctx).MintDenom`